.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     'object-property-newline': 'error',
     'import/no-named-default': 0,
     'object-curly-spacing': 0,
+    'no-extra-boolean-cast': 0,
```

Comment:
question: Was this intentional?

Reply:
Yes, it was mad about the `!!` in the conditional